frontend/packages/console-shared/src/selectors/machine.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getName } from './common';
+
+export const getMachineRole = (machine) =>
+  _.get(machine, ['metadata', 'labels', 'machine.openshift.io/cluster-api-machine-role']);
+export const getMachineNodeName = (machine) => _.get(machine, 'status.nodeRef.name');
```

Comment:
Already defined here:

https://github.com/openshift/console/blob/master/frontend/public/components/machine.tsx#L28

Reply:
I am going to update the components/machine.tsx to use selectors from @console/shared

Reply:
Done